pvlib/clearsky.py

```diff
@@ -198,13 +198,20 @@ def lookup_linke_turbidity(time, latitude, longitude, filepath=None,
     with h5py.File(filepath, 'r') as lt_h5_file:
         lts = lt_h5_file['LinkeTurbidity'][latitude_index, longitude_index]
 
+    # if localized, convert to UTC. otherwise, assume UTC.
```

Comment:
Concur with doing the conversion here. Please add a Notes section to the docstring with text like "Linke turbidity is obtained from a file of historical monthly averages. The returned value for each `time` is either the monthly value or an interpolated value to smooth the transition between months. Interpolation is done on the day of year, and for this purpose `time` is first converted to UTC."

Reply:
Thank you. I added it literally in 3fed282!

Reply:
I fixed it in another way in 4515366 to distinguish between the month problem of `lts` and the dayofyear problem of `_interpolate_turbidity` and fix the inherent problem of `_interpolate_turbidity` directly.